eng/common/TestResources/remove-test-resources.yml

```diff
@@ -3,7 +3,7 @@
 
 parameters:
   ServiceDirectory: ''
-  SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+  SubscriptionConfiguration: $(SubscriptionConfiguration)
```

Comment:
I've defaulted this because this is what deploy-test-resources outputs. It will fix the usages like we have in perf to ensure we have consistent configuration shared between deploy and remove. Should fix issue like https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4252994&view=logs&j=42ded549-05ee-5cdb-7ba7-7a948a0cc056&t=b938c548-640d-5101-7a83-c666c2d131df